erp/models.py

```diff
@@ -1001,7 +1001,7 @@ class Meta:
         null=True,
         blank=True,
         choices=schema.NULLABLE_BOOLEAN_CHOICES,
-        verbose_name="Équipement(s) sourd/malentendant",
+        verbose_name="Présence d'équipement(s) sourd/malentendant",
```

Comment:
Be careful with the plural form here. Also please use `schema.get_label("accueil_equipements_malentendants_presence")` so you define the field label at the schema level.

Reply:
Not sure about the plural here. Also, I don't get where I should use the get_label ? I could not find a similar case in the code.

Reply:
For the plural, it's simply `Présence d'équipement(s) sourds/malentendants`  instead of `Présence d'équipement(s) sourd/malentendant` (as you have in `schema.py`).

`schema.get_label` is a simple helper able to get a label from the schema. I thought it was used in models, but it's in forms. No big deal.

Reply:
Was too focused on the start of the sentence :/ Sry